### PR TITLE
Adds split video feature.

### PR DESCRIPTION
**Pull request by mbgspcii**

### CODE REVIEW

File: src/_/preCompute.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+): VideoOptions => ( {
+    "videoTransform": `${
+            from ? `from=${ from }/` : ``
+        }${
+            to ? `to=${ to }/` : ``
+        }${
+            duration ? `duration=${ duration }/` : ``
+        }`,
+    "posterTransform": `${
+            ( posterFrom || from ) ? `from=${ posterFrom || from }/` : ``
```
Comment (jaubourg) [inline src/_/preCompute.ts:17]: What happens if I use `from="4"` and `posterFrom="0"`?

Comment (mbgspcii) [reply src/_/preCompute.ts:17]: Good point. My test is clearly not accurate.

Comment (mbgspcii) [reply src/_/preCompute.ts]: - [ ] fixes posterFrom

File: src/_documentation/common/componentsProps/_propsList.md [hunk 2]
```diff
@@ -3,17 +3,21 @@
...
 | `bot` | A slash-separated list of [TwicPics API transformations](https://www.twicpics.com/docs/reference/transformations) to be performed for search engine bots. This overrides all other transformations when provided, even if empty (i.e `bot=""`). See the [TwicPics bot attribute documentation](https://www.twicpics.com/docs/reference/script-attributes#data-twic-bot) for more information. | `String` | |
+__twicVideo__| `duration` | Specifies in seconds the duration extracted from the source video. By default, `duration` is applied from the beginning of the video. You can change this behaviour by using the `from` or `to` properties. Must be a positive number as in `"10.55"` or `10.55`. See [duration documentation](https://www.twicpics.com/docs/reference/transformations#duration) for more information. | `String or number` | |
```
Comment (jaubourg) [inline src/_documentation/common/componentsProps/_propsList.md:6]: Limits the duration of the video. Duration is expressed in seconds and must be a positive number. Duration will not move the starting point of the video: to do so, you'll have to use the `from` property. See duration documentation.

File: src/_documentation/common/componentsProps/_propsList.md [hunk 3]
```diff
@@ -3,17 +3,21 @@
...
+| `focus` | Sets the focus point in `cover` mode. `focus` takes precedence over `anchor` when both are provided. See the [TwicPics focus attribute documentation](https://www.twicpics.com/docs/reference/script-attributes#data-twic-focus) for more information. Only use this attribute if you need a specific focus point or if you want to leverage smart cropping with `focus="auto"`: if you only need border-based positionning (`top`, `bottom`, `left`, `right`, etc), use `anchor` instead. | `String` | |
+__twicVideo__| `from` | Specifies in seconds the starting position of the video. By default, the video will be sliced up to the end of the source video. You can change this behaviour by using the `duration` or `to` properties. Must be a positive number as in `"5.25"` or `5.25`. See [from documentation](https://www.twicpics.com/docs/reference/transformations#from) tfor more informations. | `String or number` | |
```
Comment (jaubourg) [inline src/_documentation/common/componentsProps/_propsList.md:9]: Moves the starting point of the video. `from` is expressed in seconds and must be a positive number. `from` will not move the end point of the video: to do so, you'll have to use the `duration` or `to` properties. See from documentation.

File: src/_documentation/common/componentsProps/_propsList.md [hunk 4]
```diff
@@ -3,17 +3,21 @@
...
+| `placeholder` | Can be `preview`, `meancolor`, `maincolor` or `none`. See the [TwicPics output transformation documentation](https://www.twicpics.com/docs/reference/transformations#output) for more information. Setting will be overridden to `none` when using `zoom` `transition`. | `String` | `preview` | __TWIC_ON_STATE_CHANGE_ASSET__
 | `position` | Positions the __TWIC_SCOPE__ in `contain` mode. `position` takes precedence over `anchor` when both are provided. Syntax is the same as for CSS position properties [`background-position`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-position) and [`object-position`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position). Only use this attribute if you need precise positionning: if you only need border-based positionning (`top`, `bottom`, `left`, `right`, etc), use `anchor` instead. | `String` | `center` |
+__twicVideo__| `posterFrom` | Defines the position in seconds of the frame used to generate the poster. Must be a positive number as in `"3.05"` or `3.05`. By default, the poster is generated using the first frame of the video actually played. | `String or number` | |
```
Comment (jaubourg) [inline src/_documentation/common/componentsProps/_propsList.md:14]: Determines which frame of the source video should be used as a poster / preview. `posterFrom` is expressed in seconds and must be a positive number. By default `posterFrom` is equal to `0`, meaning the very first frame of the video is used. `posterFrom` will not modify the video in any way: to do so, you'll have to use the `duration`, `from` or `to` properties.

File: src/_documentation/common/componentsProps/_propsList.md [hunk 5]
```diff
@@ -3,17 +3,21 @@
...
 | `step` | See the [TwicPics step attribute documentation](https://www.twicpics.com/docs/reference/script-attributes#data-twic-step) for more information. | `Integer` | `10` |
 __twicImg__twicVideo__| `title` | `title` representing information related to the __TWIC_SCOPE__. See [`global attribute title`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title). | `String` | |
+__twicVideo__| `to` | Specifies in seconds the ending position of the video. By default, the video will be sliced from the very start of the source video. You can change this behaviour by using the `from` properties. Must be a positive number as in `"9.55"` or `9.55`. See [to documentation](https://www.twicpics.com/docs/reference/transformations#to) for more informations. | `String or number` | |
```
Comment (jaubourg) [inline src/_documentation/common/componentsProps/_propsList.md:20]: Moves the end point of the video. `to` is expressed in seconds and must be a positive number. `to` will not move the starting point of the video: to do so, you'll have to use the `from` property. See `to` documentation.

File: src/_documentation/common/componentsProps/_propsList.md (hunk 2 above)
Comment (mbgspcii) [reply src/_documentation/common/componentsProps/_propsList.md:6]: - [ ] update duration documentation

File: src/_documentation/common/componentsProps/_propsList.md (hunk 3 above)
Comment (mbgspcii) [reply src/_documentation/common/componentsProps/_propsList.md:9]: - [ ] update fromdocumentation

**Pull request merged**